benchopt/tests/test_benchmark_features.py

```diff
@@ -389,11 +392,10 @@ def get_data(self):
             ], standalone_mode=False)
 
         if test_case == "without_data_home":
-            out.check_output(r"path/to/data", repetition=1)
+            expected_path = re.escape(data_path)
+            out.check_output(expected_path, repetition=1)
```

Comment:
cosmetic: you can avoid introducing a new variable and just add the re.escape in checkout output; it will make the diff 1 line  shorter